src/status_im/ui/screens/accounts/login/views.cljs

```diff
@@ -19,7 +19,6 @@
             :hide-border?     true
             :title-style      {:color :white}
             :nav-action       (act/back-white #(dispatch [:navigate-back]))
-            :actions          [{:image :blank}]
```

Comment:
How come this is no longer necessary?

Reply:
it's not related to this PR, just noticed this during work on icons, and decided to fix